Bookish/Bookish.Web/Views/Home/Book.cshtml

```diff
@@ -1,7 +1,32 @@
-﻿
+﻿@model Bookish.Web.Models.BookViewModel
 @{
-    ViewData["Title"] = "Book";
+    ViewData["Title"] = Model.Book.title;
 }
 
-<h1>Book</h1>
+<h1>@ViewData["Title"]</h1>
+<p>by @Model.Book.authors</p>
 
+<div>
+    <table class="table table-striped">
+        <thead class="thead-dark">
+            <tr>
+                <th>Copy Number</th>
+                <th></th>
+            </tr>
+        </thead>
+
+        @foreach (var bookCopy in Model.Book.bookCopies)
+        {
+            <tr>
+                <td>@(Model.Book.bookCopies.FindIndex(copy => copy == bookCopy) + 1)</td>
```

Comment:
Hm, this is actually a bit annoying having to do this. I'd probably just use the copy id in that case 🙂